coding-guides/making-your-own-starboard.md

```diff
@@ -123,7 +123,8 @@ module.exports = class {
     if (message.author.id === user.id) return message.channel.send(`${user}, you cannot star your own messages.`);
     if (message.author.bot) return message.channel.send(`${user}, you cannot star bot messages.`);
     const { starboardChannel } = this.client.settings.get(message.guild.id);
-    const fetch = await message.guild.channels.find('name', starboard).fetchMessages({ limit: 100 });
+    if (!message.guild.channels.exists('name', starboardChannel)) return message.channel.send(`It appears that you do not have a \`${starboardChannel}\` channel.`); 
```

Comment:
exists() can be replaced with some(), because exists() is being deprecated soon.